api/XRSession.json

```diff
@@ -724,6 +724,55 @@
           }
         }
       },
+      "requestHitTestSource": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/XRSession/requestHitTestSource",
+          "description": "<code>requestHitTestSource({space, offsetRay})</code>",
```

Comment:
@Elchi3 Are we doing descriptions for everything now?

Reply:
No, descriptions are optional and probably not needed here.

Reply:
I can remove it :-)

It's on the other bits in this file too like requestAnimationFrame above

Reply:
Change pushed.